src/client/common/process/environmentActivationService.ts

```diff
@@ -656,7 +657,7 @@ export class EnvironmentActivationService implements IEnvironmentActivationServi
                 if (!activationCommands || activationCommands.length === 0) {
                     return;
                 }
-                traceVerbose(`Activation Commands received ${activationCommands} for shell ${shellInfo.shell}`);
+                traceInfo(`Activation Commands received ${activationCommands} for shell ${shellInfo.shell}`);
```

Comment:
Ensured to display the one message as information and the rest and versbose.
This way anytime we get the command the first we display it, and subsequent access will not be displayed in output panels (else its very noisy), but sill displayed in verbose.